### ISSUE TITLE
Missing href a11y warning

### ISSUE THREAD

**Issue by steveoh** [paulirish/lite-youtube-embed#207]
The anchor tag is displaying as an a11y warning since the href is removed.

https://github.com/paulirish/lite-youtube-embed/blob/master/src/lite-yt-embed.js#L52

Would the modern progressive enhancement approach be to start with a `<button>` that is hidden or disabled in CSS, and only show/enable it once the JS is ready to handle the click?

**Comment by davidsneighbour** [comment]
An `a` tag without `href` attribute is what is expected from a `role="button"` element. [See this official example](https://www.w3.org/WAI/ARIA/apg/patterns/button/examples/button/). This might be more of a problem of whatever parser/audit is throwing this error you are receiving.

**Comment by steveoh** [comment]
> This might be more of a problem of whatever parser/audit is throwing this error you are receiving.

It's the astro dev server tooling which I assumed was working from a modern set of rules. Your link seems to be an official doc so I'll bring up the [inconsistency with astro](https://github.com/withastro/astro/issues/15995). Thank you.
